tmail-backend/imap-extensions/src/main/resources/imapserver.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<imapservers>
+    <imapserver enabled="true">
+        <jmxName>imapserver</jmxName>
+        <bind>0.0.0.0:0</bind>
+        <connectionBacklog>200</connectionBacklog>
+        <tls socketTLS="false" startTLS="false">
+            <keystore>classpath://keystore</keystore>
+            <secret>james72laBalle</secret>
+            <provider>org.bouncycastle.jce.provider.BouncyCastleProvider</provider>
+        </tls>
+        <connectionLimit>0</connectionLimit>
+        <connectionLimitPerIP>0</connectionLimitPerIP>
+        <plainAuthDisallowed>false</plainAuthDisallowed>
+        <gracefulShutdown>false</gracefulShutdown>
+        <customProperties>pong.response=customImapParameter</customProperties>
+        <customProperties>prop.b=anotherValue</customProperties>
+        <gracefulShutdown>false</gracefulShutdown>
+        <imapPackages>com.linagora.tmail.imap.TmailImapPackage</imapPackages>
+    </imapserver>
+    <imapserver enabled="true">
+        <jmxName>imapservers</jmxName>
+        <bind>0.0.0.0:0</bind>
+        <connectionBacklog>200</connectionBacklog>
+        <tls socketTLS="true" startTLS="false">
+            <keystore>classpath://keystore</keystore>
+            <secret>james72laBalle</secret>
+            <provider>org.bouncycastle.jce.provider.BouncyCastleProvider</provider>
+        </tls>
+        <connectionLimit>0</connectionLimit>
+        <connectionLimitPerIP>0</connectionLimitPerIP>
+        <plainAuthDisallowed>false</plainAuthDisallowed>
+        <gracefulShutdown>false</gracefulShutdown>
+        <customProperties>pong.response=bad</customProperties>
+        <customProperties>prop.b=baad</customProperties>
+        <gracefulShutdown>false</gracefulShutdown>
+        <imapPackages>com.linagora.tmail.imap.TmailImapPackage</imapPackages>
```

Comment:
Call it
 
```suggestion
        <imapPackages>com.linagora.tmail.imap.TmailImapDelegationPackage</imapPackages>
```

Reply:
TmailImapPackage combines ImapDelegation and CorePackages, 
So the renaming it to TmailImapDelegationPackage may cause confusion.